homeassistant/components/nederlandse_spoorwegen/strings.json

```
@@ -0,0 +1,144 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Nederlandse Spoorwegen Setup",
```

Sentence-casing here:

```suggestion
        "title": "Nederlandse Spoorwegen setup",
```